.vscode/launch.json

```diff
@@ -18,6 +18,22 @@
         "0"
       ]
     },
+    {
```

Comment:
Let's make the `lauch.json` changes out of this PR. I assume it's not required to support `MultiMethodDecoratorFactory`.

Reply:
Will do, another PR incoming :)